xchange-poloniex2/src/main/java/info/bitrich/xchangestream/poloniex2/PoloniexStreamingMarketDataService.java

```diff
@@ -39,22 +46,21 @@ public PoloniexStreamingMarketDataService(PoloniexStreamingService service, Map<
     @Override
     public Observable<OrderBook> getOrderBook(CurrencyPair currencyPair, Object... args) {
         Observable<PoloniexOrderbook> subscribedOrderbook = service.subscribeCurrencyPairChannel(currencyPair)
```

Comment:
All three of these methods are creating duplicate fresh subscriptions to the same channel, so they are all doing the same work in `PoloniexStreamingService` and `PoloniexWebSocketEventsTransaction`.  It'd be nice if we could re-use those `service.subscribeCurrencyPairChannel` subscriptions, since all that Jackson work is quite expensive.  Could we cache them by `CurrencyPair`?

It's unrelated, so feel free to just say "maybe another time" and I'll shut up.  :)

Reply:
Yes, maybe another time ;) At least the JsonNode subscriptions are still reused using `info.bitrich.xchangestream.poloniex2.PoloniexStreamingService#subscriptions`